panorama_connector.py

```diff
@@ -1580,19 +1647,26 @@ def _run_query(self, param):
 
         return phantom.APP_SUCCESS
 
-    def _get_config_xpath(self, param):
+    def _get_config_xpath(self, param, device_entry_name=''):
+        """Return the xpath to the specified device group"""
 
+        if device_entry_name:
+            self.debug_print('Getting the Config xpath for the device entry name %s' % device_entry_name)
         device_group = self._handle_py_ver_compat_for_input_str(param[PAN_JSON_DEVICE_GRP])
 
         if device_group.lower() == PAN_DEV_GRP_SHARED:
             return '/config/shared'
 
-        return "/config/devices/entry/device-group/entry[@name='{device_group}']".format(device_group=device_group)
+        formatted_device_entry_name = ''
+        if device_entry_name:
+            formatted_device_entry_name = "[@name='{}']".format(device_entry_name)
+
+        return "/config/devices/entry{formatted_device_entry_name}/device-group/entry[@name='{device_group}']".format(
```

Comment:
can we move this big string to a constant?

Reply:
Seeing it's being used in one place, I feel putting it here can make it easy to access (i.e. we don't need to jump to another file/location to read its value. What do you think?

Reply:
As discussed, convert the long string to `DEVICE_GRP_XPATH`